packages/web-twig/src/Resources/components/Tooltip/stories/TooltipClickable.twig

```diff
@@ -10,7 +12,39 @@
     <DocsBox aria-describedby="my-js-controlled-tooltip">
         I have an externally-triggered tooltip
     </DocsBox>
-    <Tooltip id="my-js-controlled-tooltip" UNSAFE_className="is-hidden">
+    <Tooltip
+        id="my-js-controlled-tooltip"
+        UNSAFE_className="is-hidden"
+        placement="bottom-start"
+        enableControlledPlacement
```

Comment:
@pavelklibani I am not quite sure about this example because when I remove this prop the visual representation of the tip breaks. E.g. if I remember correctly both data-spirit-toggle="tooltip" and controlled placement enables floating UI functionality.

Reply:
![Screenshot 2024-01-22 at 16 49 55](https://github.com/lmc-eu/spirit-design-system/assets/2481010/384a1dd9-bcfe-4017-bcb5-b79757cf2597)

Maybe also advice from @crishpeen will be helpfull.

Reply:
not sure if understand where is the problem? to enable floating ui there 4 steps to do - `feature class`, `data-spirit-toggle="tooltip"`, `data-spirit-element="tooltip-wrapper"` and `data-placement-controlled`...  if you remove any of these 4 steps, it will be always broken i think